pkg/config/config.go

```diff
@@ -73,8 +74,9 @@ type MicroshiftConfig struct {
 	ControlPlane ControlPlaneConfig `yaml:"controlPlane"`
 	Node         NodeConfig         `yaml:"node"`
 
-	Manifests []string    `yaml:"manifests"`
-	Debug     DebugConfig `yaml:"debug"`
+	Manifests       []string             `yaml:"manifests"`
+	Debug           DebugConfig          `yaml:"debug"`
+	CSIPluginConfig lvmd.CSIPluginConfig `yaml:"csiPluginConfig"`
```

Comment:
Would it make sense to be more specific here and make the public name something like `odfLVM`? That gives us the ability to use a different name for a different plugin upstream, for example.

It's in the config file, so I think we can leave "config" out of the field name either way.

Reply:
Good point, will fix.